### ISSUE TITLE
Support ESLint 9

### ISSUE THREAD

**Issue by josundt** [JonasBa/eslint-plugin-no-lookahead-lookbehind-regexp#11]
Please migrate to support ESLint 9.

In the package.json file today, only eslint <= 8 is allowed.

**Comment by cdalton713** [comment]
Here's a quick fix until it's offically supported: 

https://github.com/cdalton713/eslint-plugin-no-lookahead-lookbehind-regexp/blob/main/package.json


package.json:
` "eslint-plugin-no-lookahead-lookbehind-regexp": "https://github.com/cdalton713/eslint-plugin-no-lookahead-lookbehind-regexp#main",`

**Comment by isuro** [comment]
@JonasBa any chance of getting this merged in/released?

**Comment by bericp1** [comment]
Yeah looks like `package.json` was updated for eslint 9 support but not published to npm yet :/

**Comment by JonasBa** [comment]
I'm going to publish this later in the day, sorry for the wait. I'm adjusting to being a dad, and this has flew under my radar, but I should do better. Will reply here once it's published

**Comment by JonasBa** [comment]
[v0.4.0](https://www.npmjs.com/package/eslint-plugin-no-lookahead-lookbehind-regexp/v/0.4.0) has now been published and should support eslint 9. I'll do a better job at keeping this up to date in the future, sorry for the long wait everyone.

**Comment by bericp1** [comment]
Thanks @JonasBa and congrats on fatherhood!
